<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Tabula Muris data â€¢ TabulaMurisData</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="Tabula Muris data">
<meta property="og:description" content="TabulaMurisData">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    

    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">TabulaMurisData</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">1.17.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../articles/TabulaMurisData.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right"></ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>Tabula Muris data</h1>
                        <h4 data-toc-skip class="author">Charlotte
Soneson</h4>
            
      
      
      <div class="hidden name"><code>TabulaMurisData.Rmd</code></div>

    </div>

    
    
<div class="section level2">
<h2 id="load-tabula-muris-data">Load Tabula Muris data<a class="anchor" aria-label="anchor" href="#load-tabula-muris-data"></a>
</h2>
<p>The <code>TabulaMurisData</code> data package provides access to the
10x and SmartSeq2 single-cell RNA-seq data sets from <a href="http://tabula-muris.ds.czbiohub.org/" class="external-link">the Tabula Muris
Consortium</a>. The contents of the package can be seen by querying the
ExperimentHub for the package name.</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/message.html" class="external-link">suppressPackageStartupMessages</a></span><span class="op">(</span><span class="op">{</span></span>
<span>    <span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va">ExperimentHub</span><span class="op">)</span></span>
<span>    <span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va">SingleCellExperiment</span><span class="op">)</span></span>
<span>    <span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va">TabulaMurisData</span><span class="op">)</span></span>
<span><span class="op">}</span><span class="op">)</span></span>
<span><span class="co">#&gt; snapshotDate(): 2022-10-24</span></span>
<span></span>
<span><span class="va">eh</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/ExperimentHub/man/ExperimentHub-class.html" class="external-link">ExperimentHub</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt; snapshotDate(): 2022-10-24</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/AnnotationHub/man/AnnotationHub-class.html" class="external-link">query</a></span><span class="op">(</span><span class="va">eh</span>, <span class="st">"TabulaMurisData"</span><span class="op">)</span></span>
<span><span class="co">#&gt; ExperimentHub with 2 records</span></span>
<span><span class="co">#&gt; # snapshotDate(): 2022-10-24</span></span>
<span><span class="co">#&gt; # $dataprovider: Tabula Muris Consortium</span></span>
<span><span class="co">#&gt; # $species: Mus musculus</span></span>
<span><span class="co">#&gt; # $rdataclass: SingleCellExperiment</span></span>
<span><span class="co">#&gt; # additional mcols(): taxonomyid, genome, description,</span></span>
<span><span class="co">#&gt; #   coordinate_1_based, maintainer, rdatadateadded, preparerclass, tags,</span></span>
<span><span class="co">#&gt; #   rdatapath, sourceurl, sourcetype </span></span>
<span><span class="co">#&gt; # retrieve records with, e.g., 'object[["EH1617"]]' </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;            title               </span></span>
<span><span class="co">#&gt;   EH1617 | TabulaMurisDroplet  </span></span>
<span><span class="co">#&gt;   EH1618 | TabulaMurisSmartSeq2</span></span></code></pre></div>
<p>The individual data sets can be accessed using either their
ExperimentHub accession number, or the convenience functions provided in
this package. For example, for the 10x data:</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">droplet</span> <span class="op">&lt;-</span> <span class="va">eh</span><span class="op">[[</span><span class="st">"EH1617"</span><span class="op">]</span><span class="op">]</span></span>
<span><span class="co">#&gt; see ?TabulaMurisData and browseVignettes('TabulaMurisData') for documentation</span></span>
<span><span class="co">#&gt; loading from cache</span></span>
<span><span class="va">droplet</span></span>
<span><span class="co">#&gt; class: SingleCellExperiment </span></span>
<span><span class="co">#&gt; dim: 23341 70118 </span></span>
<span><span class="co">#&gt; metadata(0):</span></span>
<span><span class="co">#&gt; assays(1): counts</span></span>
<span><span class="co">#&gt; rownames(23341): 0610005C13Rik 0610007C21Rik ... Zzef1 Zzz3</span></span>
<span><span class="co">#&gt; rowData names(2): ID Symbol</span></span>
<span><span class="co">#&gt; colnames(70118): 10X_P4_0_AAACCTGAGATTACCC 10X_P4_0_AAACCTGAGTGCCAGA</span></span>
<span><span class="co">#&gt;   ... 10X_P8_15_TTTGTCATCTTACCGC 10X_P8_15_TTTGTCATCTTGTTTG</span></span>
<span><span class="co">#&gt; colData names(10): cell channel ... cell_ontology_id free_annotation</span></span>
<span><span class="co">#&gt; reducedDimNames(0):</span></span>
<span><span class="co">#&gt; mainExpName: NULL</span></span>
<span><span class="co">#&gt; altExpNames(0):</span></span>
<span><span class="va">droplet</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/TabulaMurisDroplet.html">TabulaMurisDroplet</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt; see ?TabulaMurisData and browseVignettes('TabulaMurisData') for documentation</span></span>
<span><span class="co">#&gt; loading from cache</span></span>
<span><span class="va">droplet</span></span>
<span><span class="co">#&gt; class: SingleCellExperiment </span></span>
<span><span class="co">#&gt; dim: 23341 70118 </span></span>
<span><span class="co">#&gt; metadata(0):</span></span>
<span><span class="co">#&gt; assays(1): counts</span></span>
<span><span class="co">#&gt; rownames(23341): 0610005C13Rik 0610007C21Rik ... Zzef1 Zzz3</span></span>
<span><span class="co">#&gt; rowData names(2): ID Symbol</span></span>
<span><span class="co">#&gt; colnames(70118): 10X_P4_0_AAACCTGAGATTACCC 10X_P4_0_AAACCTGAGTGCCAGA</span></span>
<span><span class="co">#&gt;   ... 10X_P8_15_TTTGTCATCTTACCGC 10X_P8_15_TTTGTCATCTTGTTTG</span></span>
<span><span class="co">#&gt; colData names(10): cell channel ... cell_ontology_id free_annotation</span></span>
<span><span class="co">#&gt; reducedDimNames(0):</span></span>
<span><span class="co">#&gt; mainExpName: NULL</span></span>
<span><span class="co">#&gt; altExpNames(0):</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="explore-data-with-isee">Explore data with <code>iSEE</code><a class="anchor" aria-label="anchor" href="#explore-data-with-isee"></a>
</h2>
<p>Each data set is provided in the form of a
<code>SingleCellExperiment</code> object. To gain further insights into
the contents of the data sets, they can be explored using, e.g., the
<em><a href="https://bioconductor.org/packages/3.16/iSEE" class="external-link">iSEE</a></em>
package. For the purposes of this vignette, we first subsample a small
subset of the cells in the 10x data set, to reduce the run time.</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/Random.html" class="external-link">set.seed</a></span><span class="op">(</span><span class="fl">1234</span><span class="op">)</span></span>
<span><span class="va">se</span> <span class="op">&lt;-</span> <span class="va">droplet</span><span class="op">[</span>, <span class="fu"><a href="https://rdrr.io/r/base/sample.html" class="external-link">sample</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/seq.html" class="external-link">seq_len</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/pkg/BiocGenerics/man/nrow.html" class="external-link">ncol</a></span><span class="op">(</span><span class="va">droplet</span><span class="op">)</span><span class="op">)</span>, <span class="fl">250</span>, replace <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span><span class="op">]</span></span>
<span><span class="va">se</span></span>
<span><span class="co">#&gt; class: SingleCellExperiment </span></span>
<span><span class="co">#&gt; dim: 23341 250 </span></span>
<span><span class="co">#&gt; metadata(0):</span></span>
<span><span class="co">#&gt; assays(1): counts</span></span>
<span><span class="co">#&gt; rownames(23341): 0610005C13Rik 0610007C21Rik ... Zzef1 Zzz3</span></span>
<span><span class="co">#&gt; rowData names(2): ID Symbol</span></span>
<span><span class="co">#&gt; colnames(250): 10X_P8_12_ACGGGCTGTCAGAGGT 10X_P7_10_CGTCCATGTTATGCGT</span></span>
<span><span class="co">#&gt;   ... 10X_P7_9_TGACAACGTGTAAGTA 10X_P8_14_GATCTAGCACGGCCAT</span></span>
<span><span class="co">#&gt; colData names(10): cell channel ... cell_ontology_id free_annotation</span></span>
<span><span class="co">#&gt; reducedDimNames(0):</span></span>
<span><span class="co">#&gt; mainExpName: NULL</span></span>
<span><span class="co">#&gt; altExpNames(0):</span></span></code></pre></div>
<p>Next, we calculate size factors and normalize the data using the
<em><a href="https://bioconductor.org/packages/3.16/scran" class="external-link">scran</a></em> and
<em><a href="https://bioconductor.org/packages/3.16/scater" class="external-link">scater</a></em>
packages, and perform dimension reduction using PCA and t-SNE.</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">se</span> <span class="op">&lt;-</span> <span class="fu">scran</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/scran/man/computeSumFactors.html" class="external-link">computeSumFactors</a></span><span class="op">(</span><span class="va">se</span><span class="op">)</span></span>
<span><span class="va">se</span> <span class="op">&lt;-</span> <span class="fu">scater</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/scuttle/man/logNormCounts.html" class="external-link">logNormCounts</a></span><span class="op">(</span><span class="va">se</span><span class="op">)</span></span>
<span><span class="va">se</span> <span class="op">&lt;-</span> <span class="fu">scater</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/BiocSingular/man/runPCA.html" class="external-link">runPCA</a></span><span class="op">(</span><span class="va">se</span><span class="op">)</span></span>
<span><span class="va">se</span> <span class="op">&lt;-</span> <span class="fu">scater</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/scater/man/runTSNE.html" class="external-link">runTSNE</a></span><span class="op">(</span><span class="va">se</span><span class="op">)</span></span></code></pre></div>
<p>Finally, we call <code>iSEE</code> with the subsampled
<code>SingleCellExperiment</code> object. This opens up an instance of
<code>iSEE</code> containing the provided data set.</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw">if</span> <span class="op">(</span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">require</a></span><span class="op">(</span><span class="va"><a href="https://github.com/iSEE/iSEE" class="external-link">iSEE</a></span><span class="op">)</span><span class="op">)</span> <span class="op">{</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/pkg/iSEE/man/iSEE.html" class="external-link">iSEE</a></span><span class="op">(</span><span class="va">se</span><span class="op">)</span></span>
<span><span class="op">}</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="session-info">Session info<a class="anchor" aria-label="anchor" href="#session-info"></a>
</h2>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/sessionInfo.html" class="external-link">sessionInfo</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt; R version 4.2.2 (2022-10-31)</span></span>
<span><span class="co">#&gt; Platform: x86_64-apple-darwin17.0 (64-bit)</span></span>
<span><span class="co">#&gt; Running under: macOS Big Sur ... 10.16</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Matrix products: default</span></span>
<span><span class="co">#&gt; BLAS:   /Library/Frameworks/R.framework/Versions/4.2/Resources/lib/libRblas.0.dylib</span></span>
<span><span class="co">#&gt; LAPACK: /Library/Frameworks/R.framework/Versions/4.2/Resources/lib/libRlapack.dylib</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; locale:</span></span>
<span><span class="co">#&gt; [1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; attached base packages:</span></span>
<span><span class="co">#&gt; [1] stats4    stats     graphics  grDevices utils     datasets  methods  </span></span>
<span><span class="co">#&gt; [8] base     </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; other attached packages:</span></span>
<span><span class="co">#&gt;  [1] TabulaMurisData_1.17.0      SingleCellExperiment_1.19.1</span></span>
<span><span class="co">#&gt;  [3] SummarizedExperiment_1.27.3 Biobase_2.57.3             </span></span>
<span><span class="co">#&gt;  [5] GenomicRanges_1.49.1        GenomeInfoDb_1.33.17       </span></span>
<span><span class="co">#&gt;  [7] IRanges_2.31.2              S4Vectors_0.35.4           </span></span>
<span><span class="co">#&gt;  [9] MatrixGenerics_1.9.1        matrixStats_0.62.0         </span></span>
<span><span class="co">#&gt; [11] ExperimentHub_2.5.0         AnnotationHub_3.5.2        </span></span>
<span><span class="co">#&gt; [13] BiocFileCache_2.5.2         dbplyr_2.2.1               </span></span>
<span><span class="co">#&gt; [15] BiocGenerics_0.43.4         BiocStyle_2.25.0           </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; loaded via a namespace (and not attached):</span></span>
<span><span class="co">#&gt;   [1] Rtsne_0.16                    colorspace_2.0-3             </span></span>
<span><span class="co">#&gt;   [3] ggbeeswarm_0.6.0              ellipsis_0.3.2               </span></span>
<span><span class="co">#&gt;   [5] rprojroot_2.0.3               scuttle_1.7.4                </span></span>
<span><span class="co">#&gt;   [7] bluster_1.7.0                 XVector_0.37.1               </span></span>
<span><span class="co">#&gt;   [9] BiocNeighbors_1.15.1          fs_1.5.2                     </span></span>
<span><span class="co">#&gt;  [11] ggrepel_0.9.1                 bit64_4.0.5                  </span></span>
<span><span class="co">#&gt;  [13] interactiveDisplayBase_1.35.1 AnnotationDbi_1.59.1         </span></span>
<span><span class="co">#&gt;  [15] fansi_1.0.3                   codetools_0.2-18             </span></span>
<span><span class="co">#&gt;  [17] sparseMatrixStats_1.9.0       cachem_1.0.6                 </span></span>
<span><span class="co">#&gt;  [19] knitr_1.40                    scater_1.25.7                </span></span>
<span><span class="co">#&gt;  [21] jsonlite_1.8.3                cluster_2.1.4                </span></span>
<span><span class="co">#&gt;  [23] png_0.1-7                     shiny_1.7.3                  </span></span>
<span><span class="co">#&gt;  [25] BiocManager_1.30.19           compiler_4.2.2               </span></span>
<span><span class="co">#&gt;  [27] httr_1.4.4                    dqrng_0.3.0                  </span></span>
<span><span class="co">#&gt;  [29] assertthat_0.2.1              Matrix_1.5-1                 </span></span>
<span><span class="co">#&gt;  [31] fastmap_1.1.0                 limma_3.53.11                </span></span>
<span><span class="co">#&gt;  [33] cli_3.4.1                     later_1.3.0                  </span></span>
<span><span class="co">#&gt;  [35] BiocSingular_1.13.1           htmltools_0.5.3              </span></span>
<span><span class="co">#&gt;  [37] tools_4.2.2                   rsvd_1.0.5                   </span></span>
<span><span class="co">#&gt;  [39] igraph_1.3.5                  gtable_0.3.1                 </span></span>
<span><span class="co">#&gt;  [41] glue_1.6.2                    GenomeInfoDbData_1.2.9       </span></span>
<span><span class="co">#&gt;  [43] dplyr_1.0.10                  rappdirs_0.3.3               </span></span>
<span><span class="co">#&gt;  [45] Rcpp_1.0.9                    jquerylib_0.1.4              </span></span>
<span><span class="co">#&gt;  [47] pkgdown_2.0.6.9000            vctrs_0.5.0                  </span></span>
<span><span class="co">#&gt;  [49] Biostrings_2.65.6             DelayedMatrixStats_1.19.2    </span></span>
<span><span class="co">#&gt;  [51] xfun_0.34                     stringr_1.4.1                </span></span>
<span><span class="co">#&gt;  [53] beachmat_2.13.4               mime_0.12                    </span></span>
<span><span class="co">#&gt;  [55] lifecycle_1.0.3               irlba_2.3.5.1                </span></span>
<span><span class="co">#&gt;  [57] statmod_1.4.37                edgeR_3.39.7                 </span></span>
<span><span class="co">#&gt;  [59] scales_1.2.1                  zlibbioc_1.43.0              </span></span>
<span><span class="co">#&gt;  [61] ragg_1.2.4                    promises_1.2.0.1             </span></span>
<span><span class="co">#&gt;  [63] parallel_4.2.2                yaml_2.3.6                   </span></span>
<span><span class="co">#&gt;  [65] curl_4.3.3                    gridExtra_2.3                </span></span>
<span><span class="co">#&gt;  [67] memoise_2.0.1                 ggplot2_3.3.6                </span></span>
<span><span class="co">#&gt;  [69] sass_0.4.2                    stringi_1.7.8                </span></span>
<span><span class="co">#&gt;  [71] RSQLite_2.2.18                BiocVersion_3.16.0           </span></span>
<span><span class="co">#&gt;  [73] desc_1.4.2                    ScaledMatrix_1.5.1           </span></span>
<span><span class="co">#&gt;  [75] scran_1.25.1                  filelock_1.0.2               </span></span>
<span><span class="co">#&gt;  [77] BiocParallel_1.31.15          rlang_1.0.6                  </span></span>
<span><span class="co">#&gt;  [79] pkgconfig_2.0.3               systemfonts_1.0.4            </span></span>
<span><span class="co">#&gt;  [81] bitops_1.0-7                  evaluate_0.17                </span></span>
<span><span class="co">#&gt;  [83] lattice_0.20-45               purrr_0.3.5                  </span></span>
<span><span class="co">#&gt;  [85] bit_4.0.4                     tidyselect_1.2.0             </span></span>
<span><span class="co">#&gt;  [87] magrittr_2.0.3                bookdown_0.29                </span></span>
<span><span class="co">#&gt;  [89] R6_2.5.1                      generics_0.1.3               </span></span>
<span><span class="co">#&gt;  [91] metapod_1.5.0                 DelayedArray_0.23.2          </span></span>
<span><span class="co">#&gt;  [93] DBI_1.1.3                     pillar_1.8.1                 </span></span>
<span><span class="co">#&gt;  [95] withr_2.5.0                   KEGGREST_1.37.3              </span></span>
<span><span class="co">#&gt;  [97] RCurl_1.98-1.9                tibble_3.1.8                 </span></span>
<span><span class="co">#&gt;  [99] crayon_1.5.2                  utf8_1.2.2                   </span></span>
<span><span class="co">#&gt; [101] rmarkdown_2.17                viridis_0.6.2                </span></span>
<span><span class="co">#&gt; [103] locfit_1.5-9.6                grid_4.2.2                   </span></span>
<span><span class="co">#&gt; [105] blob_1.2.3                    digest_0.6.30                </span></span>
<span><span class="co">#&gt; [107] xtable_1.8-4                  httpuv_1.6.6                 </span></span>
<span><span class="co">#&gt; [109] textshaping_0.3.6             munsell_0.5.0                </span></span>
<span><span class="co">#&gt; [111] viridisLite_0.4.1             beeswarm_0.4.0               </span></span>
<span><span class="co">#&gt; [113] vipor_0.4.5                   bslib_0.4.0</span></span></code></pre></div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Charlotte Soneson.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.6.9000.</p>
</div>

      </footer>
</div>

  


  

  </body>
</html>
